api/python/tests/integration/test_packages.py

```diff
@@ -472,6 +474,7 @@ def test_load_into_quilt(self):
         new_pkg.set('foo2', test_file1)
 
         with patch('time.time', return_value=1234567890), \
+             patch('quilt3.packages.DISABLE_TQDM', True), patch('quilt3.data_transfer.DISABLE_TQDM', True),\
```

Comment:
Why patching is needed here?

Reply:
The reason for patching is to test for different behaviors or tqdm when disabled or enabled. In test_data_transfer, it was enabled while it is disabled here.

Reply:
Why patching is needed on this line, if you don't check outcomes of the next `push()` call? Do tests fail if you remove this line?

Reply:
I don't think it would. I'll remove it then.